# workaround for https://github.com/dyninst/dyninst/issues/406

--- dyninst-9.3.2/common/src/addrtranslate-sysv.C.sv	2017-10-03 21:32:23.608614189 -0400
+++ dyninst-9.3.2/common/src/addrtranslate-sysv.C	2017-10-03 21:33:36.538642148 -0400
@@ -800,6 +800,7 @@
          continue;
       }
       if (obj_name == "linux-vdso.so.1" ||
+	  obj_name == "linux-vdso64.so.1" ||
           obj_name == "linux-gate.so.1") 
       {
          continue;
