diff -up DyninstAPI-9.1.0/dyninstAPI_RT/h/dyninstRTExport.h.export DyninstAPI-9.1.0/dyninstAPI_RT/h/dyninstRTExport.h
--- DyninstAPI-9.1.0/dyninstAPI_RT/h/dyninstRTExport.h.export	2016-03-09 12:53:59.440594409 -0500
+++ DyninstAPI-9.1.0/dyninstAPI_RT/h/dyninstRTExport.h	2016-03-09 12:54:58.334974970 -0500
@@ -116,6 +116,7 @@ DLLEXPORT extern dyntid_t (*DYNINST_pthr
 DLLEXPORT extern unsigned int DYNINSTobsCostLow;
 DLLEXPORT extern int libdyninstAPI_RT_init_localCause;
 DLLEXPORT extern int libdyninstAPI_RT_init_localPid;
+DLLEXPORT extern int libdyninstAPI_RT_init_maxthreads;
 DLLEXPORT extern int libdyninstAPI_RT_init_debug_flag;
 DLLEXPORT extern struct DYNINST_bootstrapStruct DYNINST_bootstrap_info;
 
